tf/tfprocess.py

```diff
@@ -400,6 +400,10 @@ def init_net(self):
             self.optimizer = tf.keras.optimizers.SGD(
                 learning_rate=self.active_lr, momentum=0.9, nesterov=True)
             self.update_lr_manually = True
+            try:
```

Comment:
technically this should only be 
`self.aggregator = self.optimizer.aggregate_gradients`
because new optimizer only exists in later tf